include/eve/concept/conditional.hpp

```diff
@@ -13,7 +13,15 @@
 namespace eve
 {
   //================================================================================================
-  // Concept for conditional expression
+  //! @concept conditional_expr
+  //! @brief Specifies that a type is a Conditional Expression
+  //!
+  //! Types verifying `conditional_expr<T>` supports optimisation when evaluated where a logical
+  //! value was expected.
+  //!
+  //! @groupheader{Examples}
+  //!   - `eve::if_<eve::logical<eve::wide<char>>>`
```

Comment:
If you rly want this tp be a type, write it with `decltype`? Users should never instantiate this.

Or just say `eve::if_`

I think.

Reply:
I just put if_ so we have a  link